apps/hash-graph/libs/graph/src/snapshot/entity/batch.rs

```diff
@@ -223,46 +226,48 @@ impl<C: AsClient> WriteBatch<C> for EntityRowBatch {
             .await
             .change_context(InsertionError)?;
 
-        let entities = Read::<Entity>::read_vec(
-            postgres_client,
-            &Filter::All(Vec::new()),
-            None,
-            None,
-            None,
-            true,
-        )
-        .await
-        .change_context(InsertionError)?;
-
-        let schemas = postgres_client
-            .read_closed_schemas(&Filter::All(Vec::new()), None)
-            .await
-            .change_context(InsertionError)?
-            .try_collect::<HashMap<_, _>>()
+        if validation {
```

Comment:
This was the only change here. If you hide whitespace changes it's more readable.